### ISSUE TITLE
Investigate using Spring for OAuth2 connections

### ISSUE THREAD

**Issue by niknetniko** [ZeusWPI/hydra-android#176]
We currently use Spring for our network connections.

Maybe it would be nicer if we also used Spring for OAuth2 connections to Minerva. (Currently we do that using Apache Oltu for the initial connection, and manually after that)

**Comment by niknetniko** [comment]
No longer relevant after #258 .
